deploy/helm/templates/ingress.yaml

```diff
@@ -8,6 +8,9 @@ metadata:
     chart: {{ template "app.chart" . }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+  annotations: # this will apply whitelisting to all of the namespaces, do we only want it in stag/prod? if all connections come from Apply though they will all be in the relevant IP range so this is probably fine
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/whitelist-source-range: 35.178.209.113,3.8.51.207,35.177.252.54
```

Comment:
I suspect that this will break the smoke_tests, circle is not inside CP, but I don't care! 
The options are move ingress to the CPE repo for so we can white list prod and leave UAT as is or find a different way to run smoke tests

Reply:
I have just run into the problem of the UAT deploy not being accessible so will have to work on this a bit